### ISSUE TITLE
writer_test of serialization failed for squeeznet

### ISSUE THREAD

**Issue by WeiChungChang** [tensorflow/tensorflow#46423]
**System information**
- OS Platform and Distribution  Linux Ubuntu 16.04
- TensorFlow installed from source
- TensorFlow version (use command below):
- Python version: Python 3.5.2
- Bazel version (if compiling from source): Build label: 3.7.2
- GCC/Compiler version (if compiling from source) gcc (Ubuntu 5.4.0-6ubuntu1~16.04.12) 5.4.0 20160609

**Describe the current behavior**

- download squeezenet (squeezenet.tflit)
- build write test of serialization : bazel build -c opt //tensorflow/lite/tools/serialization:writer_test
-  bazel-bin/tensorflow/lite/tools/serialization/writer_test [model folder]/squeezenet.tflite
```
ERROR: tensorflow/lite/kernels/reshape.cc:69 num_input_elements != num_output_elements (1001 != 1)
ERROR: Node number 38 (RESHAPE) failed to prepare.

AllocateTensors failed on the round-tripped model.
```
**Describe the expected behavior**
pass write test

**Standalone code to reproduce the issue**
- download squeezenet (squeezenet.tflit)
- build write test of serialization : bazel build -c opt //tensorflow/lite/tools/serialization:writer_test
-  bazel-bin/tensorflow/lite/tools/serialization/writer_test [model folder]/squeezenet.tflite

**Other info / logs** Include any logs or source code that would be helpful to
```
ERROR: tensorflow/lite/kernels/reshape.cc:69 num_input_elements != num_output_elements (1001 != 1)
ERROR: Node number 38 (RESHAPE) failed to prepare.

AllocateTensors failed on the round-tripped model.
```
A proposal of fix is at: https://github.com/tensorflow/tensorflow/pull/46422

**Comment by abattery** [comment]
@thaink could you take a look at this issue and the proposal fix #46422 as well?

**Comment by abattery** [comment]
How did you create the model? Could you share your conversion code to reproduce and some valid sample inpus?

**Comment by WeiChungChang** [comment]
Dear @abattery
The model directly comes from tflite hosted_models zoo.
URL:
https://www.tensorflow.org/lite/guide/hosted_models
file:
https://storage.googleapis.com/download.tensorflow.org/models/tflite/model_zoo/upload_20180427/squeezenet_2018_04_27.tgz

**Comment by abattery** [comment]
Hey @WeiChungChang

I could reproduce your problem. The root cause is that, the deprecated new_shape attribute is not copied into the result of the writer library. Instead of your fix at #46422 , we will create a simple fix, instead in the writer library.

**Comment by WeiChungChang** [comment]
@abattery 
It sounds great.
Could you put the link of the patch here when close please?
I would like to know the detail and how to fix it from the other way.
Thank you!
